src/main/java/org/openstreetmap/atlas/geography/atlas/items/Relation.java

```diff
@@ -320,6 +322,30 @@ public boolean isMultiPolygon()
      */
     public abstract RelationMemberList members();
 
+    /**
+     * Get a subset of {@link #members()} matching a certain {@link ItemType}.
+     *
+     * @param itemType
+     *            - the type of members to filter.
+     * @return - {@link #members()} of type itemType.
+     */
+    public RelationMemberList membersOfType(final ItemType itemType)
```

Comment:
One suggestion for this one: perhaps the parameter could be variadic for even more flexibility?
```
public RelationMemberList membersOfType(final ItemType... itemTypes)
```

Reply:
Done!